test/test_sparse_csr.py

```diff
@@ -955,6 +957,23 @@ def test_coo_csr_conversion(self, device, dtype):
 
             self.assertEqual(csr_sparse.to_dense(), dense)
 
+    @ops(_sparse_csr_ops)
+    def test_sparse_csr_no_error(self, device, dtype, op):
+        samples = op.sample_inputs(device, dtype)
+
+        if len(samples) == 0:
+            self.skipTest("Skipped! No sample inputs!")
+
+        for sample in samples:
+            assert isinstance(sample.input, torch.Tensor)
```

Comment:
Just curious, why do we use `isinstance` here and `torch.is_tensor` below.

Reply:
I don't see a reason why you wouldn't just use ```torch.is_tensor``` in both spots.